app/api/security.py

```diff
@@ -35,6 +38,7 @@ def verify_token(token: str):
         )
         # TODO: Remove print statement or turn into logging
         print("Token verified: ", id_info)
+        return param
```

Comment:
:cherries: how about `param`-> `extracted_token` so I don't have to go check what this `get_authorization_scheme_param` is doing